packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md

```diff
@@ -233,6 +256,8 @@ RKvwyyTfqfq9pgSmB9xNVJIeVZbbZGTlNGqJti24E0AiIPggtxg5NJ+HHnEQ5RxdSsR4fbMz9i0K
 
 SAML responses are signed, and contain the profile attributes of the person who requested access to the app, as set in the General tab of the administrator UI.
```

Comment:
I'm taking the last part out. It makes absolutely no sense and not sure why it was added in the first place. 
```suggestion
SAML responses are signed and contain the profile attributes of the person who requested access to the app. 
```